.github/actions/pick/dist/index.js

```diff
@@ -0,0 +1 @@
d;t.useColors=useColors;t.storage=localstorage();t.colors=["#0000CC","#0000FF","#0033CC","#0033FF","#0066CC","#0066FF","#0099CC","#0099FF","#00CC00","#00CC33","#00CC66","#00CC99","#00CCCC","#00CCFF","#3300CC","#3300FF","#3333CC","#3333FF","#3366CC","#3366FF","#3399CC","#3399FF","#33CC00","#33CC33","#33CC66","#33CC99","#33CCCC","#33CCFF","#6600CC","#6600FF","#6633CC","#6633FF","#66CC00","#66CC33","#9900CC","#9900FF","#9933CC","#9933FF","#99CC00","#99CC33","#CC0000","#CC0033","#CC0066","#CC0099","#CC00CC","#CC00FF","#CC3300","#CC3333","#CC3366","#CC3399","#CC33CC","#CC33FF","#CC6600","#CC6633","#CC9900","#CC9933","#CCCC00","#CCCC33","#FF0000","#FF0033","#FF0066","#FF0099","#FF00CC","#FF00FF","#FF3300","#FF3333","#FF3366","#FF3399","#FF33CC","#FF33FF","#FF6600","#FF6633","#FF9900","#FF9933","#FFCC00","#FFCC33"];function useColors(){if(typeof window!=="undefined"&&window.process&&(window.process.type==="renderer"||window.process.__nwjs)){return true}if(typeof navigator!=="undefined"&&navigator.userAgent&&navigator.userAgent.toLowerCase().match(/(edge|trident)\/(\d+)/)){return false}return typeof document!=="undefined"&&document.documentElement&&document.documentElement.style&&document.documentElement.style.WebkitAppearance||typeof window!=="undefined"&&window.console&&(window.console.firebug||window.console.exception&&window.console.table)||typeof navigator!=="undefined"&&navigator.userAgent&&navigator.userAgent.toLowerCase().match(/firefox\/(\d+)/)&&parseInt(RegExp.$1,10)>=31||typeof navigator!=="undefined"&&navigator.userAgent&&navigator.userAgent.toLowerCase().match(/applewebkit\/(\d+)/)}function formatArgs(t){t[0]=(this.useColors?"%c":"")+this.namespace+(this.useColors?" %c":" ")+t[0]+(this.useColors?"%c ":" ")+"+"+e.exports.humanize(this.diff);if(!this.useColors){return}var r="color: "+this.color;t.splice(1,0,r,"color: inherit");var n=0;var o=0;t[0].replace(/%[a-zA-Z%]/g,function(e){if(e==="%%"){return}n++;if(e==="%c"){o=n}});t.splice(o,0,r)}function log(){var e;return(typeof console==="undefined"?"undefined":_typeof(console))==="object"&&console.log&&(e=console).log.apply(e,arguments)}function save(e){try{if(e){t.storage.setItem("debug",e)}else{t.storage.removeItem("debug")}}catch(e){}}function load(){var e;try{e=t.storage.getItem("debug")}catch(e){}if(!e&&typeof process!=="undefined"&&"env"in process){e=process.env.DEBUG}return e}function localstorage(){try{return localStorage}catch(e){}}e.exports=r(548)(t);var n=e.exports.formatters;n.j=function(e){try{return JSON.stringify(e)}catch(e){return"[UnexpectedJSONParseError]: "+e.message}}},984:function(e,t,r){"use strict";var n=r(228);e.exports=function settle(e,t,r){var o=r.config.validateStatus;if(!r.status||!o||o(r.status)){e(r)}else{t(n("Request failed with status code "+r.status,r.config,null,r.request,r))}}},995:function(e){"use strict";e.exports=function bind(e,t){return function wrap(){var r=new Array(arguments.length);for(var n=0;n<r.length;n++){r[n]=arguments[n]}return e.apply(t,r)}}}});
```

Comment:
does this really need to be checked in ? my js isn't so strong, but it also contains strings like
`/usr/local/google/home/sammccall/src/empty/.github/actions/pick/node_modules/last-successful-gh-commit`

Reply:
Nothing will fetch dependencies for you (here, last-successful-gh-commit as well as actions/core).

So the alternatives are checking in the whole node_modules directory, or bundling it all up into one file as we do here. Using `ncc` is recommended by the actions tutorials, it's a bit sad that it leaks the local paths in.

I added `build.sh` which produces dist/index.js. If you prefer checking in node_modules, I'm also happy to do that.

Reply:
Thanks for the offline suggestion to just run `npm`, that's much cleaner.